datum/dsl/org/openstax/ec/nudged/v1/nudged.rb

```diff
@@ -0,0 +1,13 @@
+namespace 'org.openstax.ec.nudged.v1'
```

Comment:
Might be an interesting safeguard to patch `Avro::Builder.build_dsl` to verify that the namespace actually matches the folder structure.  I dunno, maybe that's not a problem, but seems like something someone could mess up.

Reply:
i vote that this be a v2 idea